### PR TITLE
Feature/spring cleaning

### PR DESCRIPTION
**Pull request by stuartsoft**
Minor updates to python script
Updated icon assets
Dark theme
Fix for dagger errors when using databinding (https://github.com/google/dagger/issues/306)
Updated copyright and license

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -60,6 +60,11 @@ allprojects {
                     throw new GradleException("Wildcard dependency forbidden: ${requested.group}:${requested.name}:${requested.version}")
                 }
             }
+            //Fix for showing 100+ errors from javac when using Android Databinding
```
Comment (atoennis) [inline build.gradle]: I haven't noticed the errors referenced in that issue.  However, it might be worth moving the comment on lines 52 and 53 to above line 57 for clarity.

File: gradlew [hunk 2]
```diff
@@ -42,11 +42,6 @@ case "`uname`" in
     ;;
 esac
 
-# For Cygwin, ensure paths are in UNIX format before anything is touched.
```
Comment (atoennis) [inline gradlew]: Out of curiosity, what prompted the changes to gradle wrapper?

File: app/src/main/res/values/styles.xml [hunk 3]
```diff
@@ -2,10 +2,8 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+    <style name="AppTheme" parent="Theme.AppCompat.NoActionBar">
```
Comment (atoennis) [inline app/src/main/res/values/styles.xml]: The dark theme looks good but curious why the change from the current theme?

File: gradlew (hunk 2 above)
Comment (stuartsoft) [reply gradlew]: I think I opened a completely fresh Android Studio project and pasted the new `gradlew` over our starter project's `gradlew`

File: app/src/main/res/values/styles.xml (hunk 3 above)
Comment (stuartsoft) [reply app/src/main/res/values/styles.xml]: I like dark themes. That was literally the extent of my thinking :P
We can put the purple theme back in before accepting the PR if you want
or if you're ok with the dark theme, that's cool too. Whatever

File: build.gradle (hunk 1 above)
Comment (stuartsoft) [reply build.gradle]: I haven't experienced the issue either, just noticed it while browsing on Dagger's github readme.

File: app/src/main/res/values/styles.xml (hunk 3 above)
Comment (stuartsoft) [reply app/src/main/res/values/styles.xml]: If we wanna use the dark theme: no changes to the PR required
If we wanna use the purple theme: I gotta undo some theme changes
If we wanna use both: That's cool too

File: gradlew (hunk 2 above)
Comment (stuartsoft) [reply gradlew]: But to answer your question specifically, not sure why things changed. I believe cywin is a windows platform thing.... FWIW

File: build.gradle (hunk 1 above)
Comment (stuartsoft) [reply build.gradle]: Ok, got those two comments moved to lines 55 & 56 :+1:

**Comment by stuartsoft** [comment]
@atoennis

**Comment by atoennis** [comment]
Here are my thoughts:
1. Minor updates to python script - 👍 
2. Updated icon assets - 👍 
3. Dark theme - _looks good but let's leave the theme as is_
4. Fix for dagger errors when using databinding (google/dagger#306) - _Good to keep in mind but lets leave build.gradle as it was_
5. Updated copyright and license - _I don't know much about the legalities of this but_ 👍 

For 3 and 4 I'm thinking we leave leave them unchanged under the premise of fixing something only if it's an issue.  Pre-emptively fixing something might be solving a problem that doesn't exist and more code = more complexity.

Thanks for keeping things current!

**Comment by stuartsoft** [comment]
RE @atoennis 
Dark theme is gone and purple is back
build.gradle is back to normal, but I did leave the target api at 27

**Comment by stuartsoft** [comment]
@atoennis

**Comment by stuartsoft** [comment]
@atoennis @patrickhammond @weverb2

**Comment by patrickhammond** [comment]
@stuartsoft After doing some quick research, please update the license year to `2014-2018`. This reflects the year or original publish (per the commit history) through this year (where we're making changes).

Feel free to merge after that. Thanks!
